src/current/_includes/v24.1/cdc/disable-replication-ttl.md

```diff
@@ -13,10 +13,7 @@ CREATE TABLE tbl (
 ALTER TABLE events SET (ttl_expire_after = '1 year', ttl_disable_changefeed_replication = 'true');
 ~~~
 
-You can also widen the scope of disabling changefeed replication in a cluster by using one of the following:
-
-- `sql.ttl.changefeed_replication.disabled` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}): when `true`, deletes issued by TTL jobs on the cluster will not be emitted by changefeeds.
-- `disable_changefeed_replication` [session variable]({% link {{ page.version.version }}/set-vars.md %}): when `true`, changefeeds will not emit messages for any changes (e.g., `INSERT`, `UPDATE`) issued to watched tables during that session.
```

Comment:
I removed this session variable from this include file, so that it does not pull into the row level ttl page.